package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@cdssnc/gcds-fonts",
+  "version": "0.0.1",
```

Comment:
Are we releasing it first as `0.0.1` or should we do a full release `1.0.0`?

Reply:
I added it as `0.0.1` so we can do a test release and there is one more change coming up to the tokens. I'll switch to `1.0.0` after that, but yes, definitely want to do a full release right away for this package.